src/components/custom-field-input-multiple-choice/custom-field-input-multiple-choice.test.jsx

```diff
@@ -16,18 +25,80 @@ describe('<CustomFieldInputMultipleChoice>', () => {
     expect(renderer.create(<CustomFieldInputMultipleChoice {...requiredProps} />).toJSON()).toMatchSnapshot();
   });
 
+  describe('autocompleter popup', () => {
```

Comment:
```suggestion
  describe('listbox popup', () => {
```

Just so we don't confuse ourselves when we add the `autocomplete` functionality.

Reply:
It's part of the autocomplete functionality though. The popup are the values for the autocomplete!